### PR TITLE
Add Submodule for `pw-link`

### PR DESCRIPTION
**Pull request by engineerjoe440**
### Updates:

* Touch up README.md
* `python` --> `Python`
* Update `pyproject.toml` with common attributes

### Additions:
* Add `link.py` submodule to support `pw-link`
* Add basic tests for link submodule

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -2,25 +2,32 @@
 requires = ["flit_core >=2,<4"]
 build-backend = "flit_core.buildapi"
 
-[tool.flit.metadata]
-module = "pipewire_python"
+[project]
+name = "pipewire_python"
 author = "Pablo Diaz"
```
Comment (pablodz) [inline pyproject.toml]: I think its time to change to pipewire_python's authors

**Comment by pablodz** [comment]
Also, please change the version  to 0.2.2 

https://github.com/pablodz/pipewire_python/blob/01a38f185735b90cce51dbf46f37c37b738a8422/pipewire_python/__init__.py#L70-L70

File: pyproject.toml (hunk 1 above)
Comment (engineerjoe440) [reply pyproject.toml]: Fair point! I've updated to the listed dictionaries style that seems available for Flit and Poetry.

@annaabsi, @kwint, @mrteathyme: I presently have you listed as authors for your contributions. But I'd like to confirm your willingness to be listed here (don't want to put your names "out there" if you don't feel comfortable with it).

**Comment by engineerjoe440** [comment]
> Also, please change the version to 0.2.2
> 
> https://github.com/pablodz/pipewire_python/blob/01a38f185735b90cce51dbf46f37c37b738a8422/pipewire_python/__init__.py#L70-L70

Definitely! I wasn't thinking of that.

File: pyproject.toml (hunk 1 above)
Comment (pablodz) [reply pyproject.toml]: maybe only "pipewire_python's authors"

Comment (kwint) [reply pyproject.toml]: Thanks for asking, I rather not be listed as an author. I'm fine with being listed as a contributor, but author sounds like to much recognition for my 5 lines of code and I don't think I will add much to this repo in the future

**Comment by pablodz** [comment]
@engineerjoe440 the PR is still as draft, tag me when it's ready please

**Comment by engineerjoe440** [comment]
> @engineerjoe440 the PR is still as draft, tag me when it's ready please

Sure, @pablodz ! I'll get my branch updated for authors.

There's still a point about PRs targeting a `develop` branch, did you want me to re-target this PR?

**Comment by pablodz** [comment]
I'ts ok to target main , I'll run some tests before releasing to pypi

**Comment by engineerjoe440** [comment]
Thanks, @pablodz!

I can't quite make out what's erroring out in the build tests, so if there are things you would like me to address, just let me know! :smile:

Thank you!

**Pull request merged**